### ISSUE TITLE
[Enhancement] Notifications

### ISSUE THREAD

**Issue by UltraStudioLTD** [netxs-group/vtm#227]
While I added this to #183 (**App Roadmap**), I think it will be better to create issue addressing this.

I was looking for rices on **https://reddit.com/r/unixporn** and I accidentally stumbled upon one [**NeoViM**](https://github.com/neovim/neovim) screenshot which had nice notification from right side (like how it is on **Microsoft Visual Studio Code**, **Windows** or most of Desktop Environments. After researching I found out that it was created using [**nvim-notify**](https://github.com/rcarriga/nvim-notify) (Below is example)
![nvim-notify example](https://user-images.githubusercontent.com/24252670/130856848-e8289850-028f-4f49-82f1-5ea1b8912f5e.gif)

So I thought that it would be great to create something similar for **vtm** with certain API so I created this issue.

What do you think @o-sdn-o? It even has few appear-disappear styles: **slide [w/o fade]**, **fade**, **static**

**Comment by o-sdn-o** [comment]
Yes, of course this is a necessary feature. This can be implemented in the spirit https://gitlab.freedesktop.org/terminal-wg/specifications/-/issues/13

**Comment by UltraStudioLTD** [comment]
Also project mentioned was inspired by this https://neovim.discourse.group/t/wip-animated-notifications-plugin/448 showcase which shows better notifications (for example look at clock). After digging around this is repo of original author (currently notify is not implemented but some other cool things are) [stylish.nvim](https://github.com/sunjon/stylish.nvim). I think it's worth checking it out and getting some cool things from it.

Here is original inspiration material:
![Updated GIF](https://aws1.discourse-cdn.com/free1/uploads/neovim/original/1X/9a74c88fe3207395988acc6f135742d0b2c16347.gif)

With little style addition:
![Other style](https://aws1.discourse-cdn.com/free1/uploads/neovim/original/1X/fee95effe1d3621fac68746fe92e925fe2651a1c.gif)
